assets/js/modules/adsense/components/dashboard/DashboardTopEarningPagesWidget.js

```diff
@@ -85,11 +85,11 @@ function DashboardTopEarningPagesWidget() {
 	}
 
 	if ( error ) {
-		return getDataErrorComponent( 'analytics', error.message, false, false, false, error );
+		return <ReportError moduleSlug="adsense" error={ error } />;
```

Comment:
This should remain `analytics` as it was before
```suggestion
		return <ReportError moduleSlug="analytics" error={ error } />;
```